runtime/lua/vim/_meta/api.lua

```diff
@@ -425,6 +425,7 @@ function vim.api.nvim_buf_get_number(buffer) end
 --- @return integer
 function vim.api.nvim_buf_get_offset(buffer, index) end
 
+--- @deprecated
```

Comment:
This file is generated.

Reply:
Right, but the generator is wrong (?)

Reply:
Looks like it. _Another_ case where we need to fix the generator 😉

Reply:
AHHHH
I’ll try at least fixing this for this PR.